public/js/components/Editor.css

```diff
@@ -76,7 +76,7 @@
 }
 
 .welcomebox {
-  width: 100%;
+  width: calc(100% - 1px);
```

Comment:
this is a super strange issue w/ how the splitter handle is bigger than the splitter line.